src/main/java/DialogBox.java

```diff
@@ -0,0 +1,46 @@
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+
+import javafx.geometry.Pos;
+
+import javafx.scene.Node;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.HBox;
+
+public class DialogBox extends HBox {
```

Comment:
Actually popup windows can also be written in FXML and be loaded using an FXML loader.